### PR TITLE
Add links to the JWebbinar notebooks and videos to the docs

### PR DESCRIPTION
**Pull request by rosteen**
Simple enough!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -121,6 +121,11 @@ If you uncover any issues or bugs, you can
 if they are not already reported. For faster responses, however, we encourage you to
 submit a `JWST Help Desk Ticket <https://jwsthelp.stsci.edu>`_.
 
+Recordings and instructional notebooks from live Jdaviz tutorials (as part of the JWebbinars series)
+can be found at `the JWebbinar website <https://www.stsci.edu/jwst/science-execution/jwebbinars>`_
+under the "Materials and Videos" expandable section. Scroll down to the bottom of that section to
+find materials from the most recent session (JWebbinar 24).
```
Comment (pllim) [inline README.rst:127]: "recent" won't be true anymore as time goes on. I propose we replace this with the actual date.

File: docs/index.rst [hunk 2]
```diff
@@ -34,6 +34,13 @@ contextual information like on-sky views of the spectrograph slit.
    The offline version of this documentation can be downloaded from
    `Jdaviz zipped HTML downloads page <https://readthedocs.org/projects/jdaviz/downloads/>`_.
 
+.. note::
+
+   Recordings and instructional notebooks from live Jdaviz tutorials can be found at
+   `the JWebbinar website <https://www.stsci.edu/jwst/science-execution/jwebbinars>`_
+   under the "Materials and Videos" expandable section. Scroll down to the bottom of that section
+   to find materials from the most recent session (JWebbinar 24).
```
Comment (pllim) [inline docs/index.rst:42]: Same comment as above.

File: README.rst (hunk 1 above)
Comment (rosteen) [reply README.rst]: Well, it does say _most_ recent. But I went ahead and added dates.

Comment (pllim) [reply README.rst]: If we do another one and forget to update the docs, then... 😬

**Comment by pllim** [comment]
Most of the dev failures might just go away after astropy patches numpy dev compatibility.

**Review by pllim — APPROVED** [review]
Doc looks like doc. Thanks!

**Review by javerbukh — APPROVED** [review]

**Pull request merged**